server/resources/review_respond.py

```diff
@@ -0,0 +1,55 @@
+from flask_restful import Resource, reqparse
+from flask_jwt_extended import (jwt_required, get_jwt_identity)
+from models.review_model import ReviewModel
+from extensions import queue, create_app
+from tasks.find_reviewer_task import find_reviewer
+from datetime import timedelta
+
+
+class ReviewRespond(Resource):
+    @jwt_required
+    def post(self):
+
+        parser = reqparse.RequestParser()
+        parser.add_argument(
+            "review_id", help="This field cannot be blank", required=True)
+        data = parser.parse_args()
+
+        # get review
+        user_id = get_jwt_identity()
+        review = ReviewModel.get_review(data["review_id"])
+
+        # update review status to in_review
+        ReviewModel.update_status(review.id, "in_review")
```

Comment:
You probably want to do some validation that user_id is the reviewer in the ReviewModel